2020/13xxx/CVE-2020-13663.json

```diff
@@ -1,18 +1,83 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'vendor_name': 'Drupal', 'product': {'product_data': [{'product_name': 'Drupal Core', 'version': {'version_data': [{'version_name': '7.x', 'version_affected': '<', 'version_value': '7.72', 'platform': ''}, {'version_name': '8.8.x', 'version_affected': '<', 'version_value': '8.8.8', 'platform': ''}, {'version_name': '8.9.x', 'version_affected': '<', 'version_value': '8.9.1', 'platform': ''}, {'version_name': '9.0.x', 'version_affected': '<', 'version_value': '9.0.1', 'platform': ''}]}}]}}]}}